vendor/paella_overrides/plugins/es.upv.paella.rtmpPlayer/rtmp_player.js

```diff
@@ -128,10 +128,14 @@ Class ("paella.RTMPVideo", paella.VideoElementBase,{
 
 			var text = document.createElement('div');
 
-			text.innerHTML = base.dictionary.translate("You need at least Flash player 9 installed.") + "<br>" +
-				base.dictionary.translate("Please go to {0} and install it.")
-					.replace("{0}", "<a style='color: #800000; text-decoration: underline;' href='http://www.adobe.com/go/getflash'>http://www.adobe.com/go/getflash</a>");
-
+                        // #DCE MATT-2429-Change-Flashdownload-text-ifChrome-1x
+                        if (base.userAgent.browser.Name === "Chrome") {
+			  text.innerHTML = base.dictionary.translate("You need to enable Flash to watch this.") + "<br>" +
+					"<a style='color: #800000; text-decoration: underline;' href='http://www.adobe.com/go/getflash'>Click for Enable</a>";
+                        } else {
+			  text.innerHTML = base.dictionary.translate("You need to enable or download Flash to watch this.") + "<br>" +
+                                        "<a style='color: #800000; text-decoration: underline;' href='http://www.adobe.com/go/getflash'>Click for Enable or Download</a>";
```

Comment:
Just for consistency, I'd still go with Chrome's language:
"Click to enable or download Adobe Flash Player"

Reply:
done!